doc/changelog.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 Version 0.5.6
 -------------
 
+* Change ``warn`` to ``warning`` in `bm_logging.py` .In Python 2, Logger.warn()
```

Comment:
No, it's only done in Python 3. Please mention that warnings.warn() is called to log a deprecation warning.